supervisord.conf

```diff
@@ -0,0 +1,42 @@
+; Sample supervisor config file.
```

Comment:
I modified sample config file according to our needs. Forgot to remove that comment